File: Ruby/ListItem.rb
class ListItem
	def initialize(value)
		@value = value
		@nextNode
	end

	def getValue()
		return @value
	end

	def setValue(value)
		@value = value
	end

	def getNextNode()
		return @nextNode
	end

	def setNextNode(node)
		@nextNode = node
	end
end

File: Python2.7/ListItem.py
class ListItem:

	def __init__(self, value):
		self.value = value

	def getValue(self):
		return self.value

	def setValue(self, value):
		self.value = value

	def getNextItem(self):
		return self.nextItem

	def setNextItem(self, nextItem):
		self.nextItem = nextItem